ui/src/styles.sass

```diff
@@ -1 +1,5 @@
 /* You can add global styles to this file, and also import other style files */
+body
+  margin: 0%
+
+@import 'material-icons/iconfont/material-icons.scss'
```

Comment:
the import line must be on the first of the file

Reply:
ok